resources/public/api/conf/2.0/schemas/retrieve_pension_charges_response.json

```diff
@@ -94,6 +94,14 @@
                   "type": "string",
                   "example": "Q123456"
                 }
+              },
+              "pensionSchemeTaxReference":{
+                "type":"array",
+                "items": {
+                  "description":"A list of one or more unique references provided by HM Revenue and Customs (HMRC) when the scheme has been registered for tax relief and exemptions. It is 10 characters made up of 8 numbers followed by 2 letters.",
+                  "type": "string",
+                  "example": "00123456RA"
+                }
```

Comment:
Can't comment on specific line, but Line 185. This field is optional, not supposed be under `required`.